### PR TITLE
Add delay to pipeline if another build is running

### PR DESCRIPTION
**Pull request by thomast1906**
### Change description ### 
Add delay to pipeline if another build is running

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: azure_pipeline.yaml [hunk 1]
```diff
@@ -13,7 +13,62 @@ variables:
...
+          vmImage: $(agentPool)
+        steps:
+        - task: AzureKeyVault@1
+          inputs:
+            ConnectedServiceName: azurerm-sandbox
+            keyVaultName: infra-vault-nonprod
+            secretsFilter: 'azure-devops-token'
+            runAsPreJob: false # Azure DevOps Services only
+        - task: Bash@3
+          env:
```
Comment (timja) [inline azure_pipeline.yaml:29]: why'd you open a new PR? the old one had a comment requesting a display name here as a form of documenting what this is doing

Comment (thomast1906) [reply azure_pipeline.yaml:31]: Updated both tasks with a displayName, closed other as was spamming it with files changed to testing. It had a stale entry for notCompleted so how I was doing that in that PR was invalid. Felt a new PR was required.

Comment (timja) [reply azure_pipeline.yaml:31]: just change your current PR, people lose context and explanations on the previous PR are now gone, also frustrates people when they left a review on the previous one and then it's not there on the new one and comments aren't addressed

Comment (thomast1906) [reply azure_pipeline.yaml:31]: Sure, will close and reopen old with new changes.

**Pull request closed without merging**